sunxspex/thermal_spectrum.py

```diff
@@ -49,6 +49,21 @@ def __init__(self, energy_edges,
             Cannot be set is observer_distance kwarg also set.
             Default=None.
 
+        Notes
+        -----
+        The default CHIANTI data used here is contained within the `sunpy.data.manager` and is collected
+        from `https://hesperia.gsfc.nasa.gov/ssw/packages/xray/dbase/chianti/chianti_lines_1_10_v71.sav`.
+        If the user would like to use a different file, the default can be overwritten using the context 
+        manager `sunpy.data.manager`. 
+        For example:
+        >>> import numpy as np
+        >>> from astropy import units as u
+        >>> from sunxspex import thermal_spectrum
+        >>> from sunpy.data import manager
+        >>> energy_bins = np.arange(3, 100, 0.5)*u.keV
+        >>> my_file = "https://hesperia.gsfc.nasa.gov/ssw/packages/xray/dbase/chianti/chianti_lines_1_10_v70.sav"
+        >>> with manager.override_file("chianti_lines_1_10", uri=my_file):
```

Comment:
I think also you should skip this in the doc tests.  Add ```# doctest: +SKIP``` to the end of this line and the next.